modules/import-export/mod-pcm/ImportPCM.cpp

```diff
@@ -289,15 +289,15 @@ void PCMImportFileHandle::Import(
 
    wxASSERT(mFile.get());
 
-   auto track = ImportUtils::NewWaveTrack(
+   auto trackList = ImportUtils::NewWaveTracks(
```

Comment:
`ImportRAW.cpp` directly invokes factory's `CreateMany`, would be nice to keep it same in both.